src/main/java/bookup/SQLHelper.java

```diff
@@ -1,6 +1,87 @@
 package bookup;
+import javax.xml.transform.Result;
+import java.sql.*;
+import java.util.*;
 
 /** Provides helper methods for interacting with the database */
 class SQLHelper {
-	
+
+    /** Adds specified book to database */
+    static void add(Book book) {
+        Connection con = connect();
+        try {
+            Statement statement = con.createStatement();
+            String SQLAddStatement = String.format("insert into books values(%s, '%s', '%s', %s, %s, '%s');", book.getISBN(), book.getTitle(), book.getAuthor(),book.getPages(), book.getYear(), book.getImageURL());
+            statement.executeUpdate(SQLAddStatement);
+            con.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /** Returns a Book with the specified isbn or null if the book does not exist in the database */
+    static Book getBook(String isbn) {
+        Connection con = connect();
+        ResultSet results;
+        Book book = null;
+        try {
+            Statement statement = con.createStatement();
+            results = statement.executeQuery(String.format("select * from books where isbn = %s;", isbn));
+            book = createBook(results);
+            con.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return book;
+    }
+
+    /** Returns a List of books by the specified author */
+    static List<Book> getBooks(Author author){
```

Comment:
why not by ISBN?

Reply:
actually I dont see the benefit of either. Maybe just a getAllBooks()?

Reply:
Because there won't ever be more than one book with the same isbn. I can write getAllBooks too.

Reply:
I think we only need getAllBooks() or listBooks(). I dont see someone needing to get a book by an author. You can leave for now, but definitely need getAllBooks()